Userland/Libraries/LibGUI/Desktop.h

```diff
@@ -39,7 +39,7 @@ class Desktop {
     unsigned workspace_rows() const { return m_workspace_rows; }
     unsigned workspace_columns() const { return m_workspace_columns; }
 
-    int taskbar_height() const { return TaskbarWindow::taskbar_height(); }
+    int taskbar_height() const { return Taskbar::TaskbarWindow::taskbar_height(); }
```

Comment:
If you want to change this name namespace, it probably makes sense to rename `TaskbarWindow` =>  `Window`, so then we get the nice `Taskbar::Window` type name instead.